packages/salesforcedx-vscode-apex-replay-debugger/test/unit/adapter/debugConfigurationProvider.test.ts

```diff
@@ -68,8 +68,8 @@ describe('Configuration provider', () => {
     const config = DebugConfigurationProvider.getConfig('/path/foo.cls');
     expect(config).to.deep.equal(expectedConfig);
   });
-
-  it('Should fill in empty attributes in the config', () => {
+  /*
```

Comment:
Now that `debugConfigurationProvider.ts` is directly calling jorje I need to figure out whether I create some mocks to keep this as close to being a unit test as possible or go and write an integration test. Would like to know what you think.